### ISSUE TITLE
Reply to address admin notification should be the address of the submitter

### ISSUE THREAD

**Issue by noregt** [am-impact/amforms#41]
I noticed that the reply address of the admin notification mail is standard set in the form notification tab.
This results in the admin (my client) answering to his/her own address. The address should be the submitters address (toEmail?)
For the submitters' copy the reply address should be the set reply address in the notification tab or the (fromEmail).

**Comment by noregt** [comment]
Partly solved this by setting {mailfieldHandle} as replyTo mail address in the notification. Drawback is that I had to disable the copy for the submitter. Would be ideal if there was a dedicated replyTo field for the notification copy.

**Comment by hubertprein** [comment]
@noregt Yeah, there have been more requests for more customisation on the notifications end. I have to create something for that, but unfortunately, I have no release date for that right now.

**Comment by noregt** [comment]
Okay, thanks for the information Ruben

**Comment by kaspar-allenbach** [comment]
+1

**Comment by kaspar-allenbach** [comment]
Hi. I thinking about switching to a&m forms on several sites because a&m Forms is by far the best form plugin for craft imho.

One of my dev colleages wrote a notification plugin for the standard p&t form plugin. Do you use the `beforeSend` event? Because if you do we could move the plugin to work for a&m forms as well.

**Comment by robzor** [comment]
@kaspar-allenbach that would be great :)

**Comment by hubertprein** [comment]
@kaspar-allenbach Our plugin uses its own events, so it's not triggering on ```beforeSend``` right now. Am I right to assume you want support for this event? Who's event is this? Because it's not in Craft's core.

**Comment by kaspar-allenbach** [comment]
@hubertprein No I'd like to have this issue solved right in your plugin. Why have a plugin who builds on another plugin. The before send event comes from the p&t form plugin then. I thought that would be a craft feature.

**Comment by hubertprein** [comment]
@kaspar-allenbach Fair enough. Then for now I'll have to refer to my answer given earlier regarding this topic [here](https://github.com/am-impact/amforms/issues/41#issuecomment-168668550).

**Comment by lindseydiloreto** [comment]
Looks like this thread got hijacked, but I want to +1 the original concerns by @noregt... It would be extremely helpful if my client could hit "Reply" to actually reply to the person submitting the form.

**Comment by hubertprein** [comment]
@lindseydiloreto You could do this: http://cl.ly/1z2n2u453E2t

**Comment by lindseydiloreto** [comment]
Thanks @hubertprein, that's exactly what we ended up doing. Feels more like a workaround than a proper solution though, correct? Or is this actually the official way to do it?

**Comment by noregt** [comment]
@hubertprein I have a script that sends a copy to the submitter and sets the submitters reply address for the admin. If you want I can send it. (uses $email->fromEmail = $email->replyTo)

**Comment by lindseydiloreto** [comment]
@noregt Can you post a gist of that solution? For whatever reason, the previous workaround doesn't seem to be working any more. :(

Thanks!!

**Comment by kaspar-allenbach** [comment]
@noregt I'd like to have this script as well :) :+1:

**Comment by noregt** [comment]
This script works with the regular contactform from P&T (so NOT amforms), still pretty useful I hope
https://gist.github.com/noregt/84910bdf647d23e0c752

**Comment by raymondelooff** [comment]
@hubertprein I think that this issue can be closed now. Version 1.6.0 added this functionality :)
